packages/modelviewer.dev/examples/stagingandcameras/index.html

```diff
@@ -247,8 +247,8 @@ <h4>Event listeners can cooperate with camera-controls</h4>
         Math.abs(pointer.clientY - startY) > tapDistance)
       return;
     const rect = modelViewer.getBoundingClientRect();
-    const x = event.clientX - rect.left;
-    const y = event.clientY - rect.top;
+    const x = pointer.clientX - rect.left;
```

Comment:
Lack of type-checking bit us again. Surprisingly, the browser injects `event` as a variable in the event handler scope.
Try:
`window.addEventListener('click', () => console.log(event))`

It was working with mouse events because unlike `TouchEvent`, `MouseEvent` has properties `clientX` and `clientY`.